### ISSUE TITLE
Creating a new Angular-based workspace from scratch fails on NX11

### ISSUE THREAD

**Issue by jmarlow4** [nrwl/nx#4265]
## Current Behavior

running `npx create-nx-workspace@latest nxm` and select Angular, this results in "generator workflow failed" error, doesn't matter what kind of styling or linting you pick. The error that causes it is `npm ERR! notarget No matching version found for @angular-eslint/eslint-plugin@11.0.0.`

## Expected Behavior

Create the NX workspace (that works) and then add Angular project app and install deps

This is a regression. This doesn't happen when you run it without the `@latest` suffix. But if you do that, it installs some packages then gives me another set of errors:
`npm ERR! ERESOLVE unable to resolve dependency tree`
`npm ERR!`
`npm ERR! While resolving: nxm10@0.0.0`
`npm ERR! Found: @angular/compiler@10.2.3`
`npm ERR! node_modules/@angular/compiler`
`npm ERR!   @angular/compiler@"^10.1.0" from the root project`
`npm ERR!`
`npm ERR! Could not resolve dependency:`
`npm ERR! peer @angular/compiler@">=2.3.1 <9.0.0 || >8.0.0-beta <9.0.0" from codelyzer@5.0.1`
`npm ERR! node_modules/codelyzer`
`npm ERR!   dev codelyzer@"~5.0.1" from the root project`

## Steps to Reproduce

Just execute the command.

### Can you reproduce this on https://github.com/nrwl/nx-examples?
Yes, when I clone the examples repo and run `npm install` or even with the `--force` flag, I get the following:
`npm ERR! notarget No matching version found for @angular-eslint/eslint-plugin@11.0.0.`
`npm ERR! notarget In most cases you or one of your dependencies are requesting`
`npm ERR! notarget a package version that doesn't exist.`

### Environment

Using Node 15.3.0 and npm v 7.0.14 on MacOS Big Sur

**Comment by FrozenPandaz** [comment]
Hi, I see the issue, we have an improper `peerDependency`. I will remove it.

However, Is there some setting where `npm` will fail when it can't resolve the version of a `peerDependency`? I don't get an error when creating a new workspace or on `nx-examples`.

**Comment by jmarlow4** [comment]
That's what I'm wondering. Something like this is so basic.. I'm trying to figure out right now if it just might be my setup

**Comment by jmarlow4** [comment]
OK so I think I figured it out. And I'm pretty sure this is it. NPM version 7 doesn't play nice with NX for some reason. I bumped my version of node down to 14 for a larp, and it brought down the NPM version to 6.14.8. Retried the same commands and it worked without a hitch. So I bumped Node back up to 15.3 and just installed npm@6.14.8 globally to replace 7.0.14, retried the npx commands to generate the workspace and angular app, and it worked just fine.

I think this issue is resolved
